static/css/partials/_animations.scss

```diff
@@ -1,19 +1,15 @@
 .loader {
-  background: #2e2d30;
+  background: rgba(12, 12, 13, 0.2);
```

Comment:
this could be a variable, but we can just fix this here: https://github.com/mozilla-services/screenshots/issues/3864

I'm reluctant to add more color vars until we just update everything to be on the photon design system